### ISSUE TITLE
Remove link in footer to  'View the showcase'? (h5bp.net)

### ISSUE THREAD

**Issue by coliff** [h5bp/html5boilerplate.com#156]
The link in the footer to  'View the showcase' goes to: http://h5bp.net/ which hasn't been updated for more than a year.

@pestbarn - are you still maintaining this site?

**Comment by pestbarn** [comment]
Yeah, I'm still maintaining it. However, I haven't received a single submission for over a year now, hence the site not being updated.
